packages/mobile/src/identity/verification.ts

```diff
@@ -64,6 +68,13 @@ export interface AttestationCode {
   issuer: string
 }
 
+export function* isSufficientBalance() {
+  const e164Number: string = yield select(e164NumberSelector)
+  const cachedSalt = yield call(getSaltCache, e164Number)
+  const isSufficientForQuota = cachedSalt ? true : yield call(balanceSufficientForQuotaRetrieval)
+  yield put(setVerificationPossible(isSufficientForQuota))
```

Comment:
It seems like we didn't need new state and a new watcher to compute this. We could have used a reselect selector to compute it since it's purely derived from other state.